src/test/java/org/springframework/data/hazelcast/repository/query/HazelcastSortAccessorTest.java

```diff
@@ -0,0 +1,146 @@
+package org.springframework.data.hazelcast.repository.query;
+
+import com.hazelcast.query.Predicate;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.history.RevisionSort;
+import org.springframework.data.keyvalue.core.query.KeyValueQuery;
+import org.springframework.util.ObjectUtils;
+
+import java.io.Serializable;
+import java.util.*;
+
+import static org.junit.Assert.*;
+
+public class HazelcastSortAccessorTest {
+
+    @Rule
+    public ExpectedException exceptionRule = ExpectedException.none();
+
+    @Test()
+    public void test1() {
+        exceptionRule.expect(UnsupportedOperationException.class);
+        exceptionRule.expectMessage("Null handling not implemented:");
+        KeyValueQuery<String> keyValueQuery0 = new KeyValueQuery<String>("^pg~]=P");
+        Sort.Order[] sort_OrderArray0 = new Sort.Order[7];
+        Sort.Order sort_Order0 = Sort.Order.asc("^pg~]=P");
+        sort_OrderArray0[0] = sort_Order0;
+        Sort.Order sort_Order1 = sort_Order0.nullsLast();
+        sort_OrderArray0[1] = sort_Order1;
+        Sort sort0 = Sort.by(sort_OrderArray0);
+        keyValueQuery0.setSort(sort0);
+        HazelcastSortAccessor hazelcastSortAccessor0 = new HazelcastSortAccessor();
+        hazelcastSortAccessor0.resolve(keyValueQuery0);
+    }
+
+    @Test
+    public void test2() {
+        exceptionRule.expect(UnsupportedOperationException.class);
+        exceptionRule.expectMessage("Ignore case not implemented");
+        HazelcastSortAccessor hazelcastSortAccessor0 = new HazelcastSortAccessor();
+        RevisionSort revisionSort0 = RevisionSort.asc();
+        KeyValueQuery<Object> keyValueQuery0 = new KeyValueQuery<Object>((Sort) revisionSort0);
+        Sort.Order[] sort_OrderArray0 = new Sort.Order[4];
```

Comment:
Could you stick to Java coding conventions and use camel case instead of underscores?

Reply:
yes, corrected